[2022-05-31 20:09:49,198] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: todoist_load_to_pg.run_this_first manual__2022-05-31T20:09:47.572099+00:00 [queued]>
[2022-05-31 20:09:49,212] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: todoist_load_to_pg.run_this_first manual__2022-05-31T20:09:47.572099+00:00 [queued]>
[2022-05-31 20:09:49,213] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-31 20:09:49,214] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-05-31 20:09:49,215] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-05-31 20:09:49,232] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): run_this_first> on 2022-05-31 20:09:47.572099+00:00
[2022-05-31 20:09:49,239] {standard_task_runner.py:52} INFO - Started process 115 to run task
[2022-05-31 20:09:49,243] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'todoist_load_to_pg', 'run_this_first', 'manual__2022-05-31T20:09:47.572099+00:00', '--job-id', '373', '--raw', '--subdir', 'DAGS_FOLDER/todoist_load_to_pg.py', '--cfg-path', '/tmp/tmpb9sou23d', '--error-file', '/tmp/tmpi5tys92w']
[2022-05-31 20:09:49,245] {standard_task_runner.py:77} INFO - Job 373: Subtask run_this_first
[2022-05-31 20:09:49,307] {logging_mixin.py:109} INFO - Running <TaskInstance: todoist_load_to_pg.run_this_first manual__2022-05-31T20:09:47.572099+00:00 [running]> on host bd658c040114
[2022-05-31 20:09:49,362] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=emil
AIRFLOW_CTX_DAG_ID=todoist_load_to_pg
AIRFLOW_CTX_TASK_ID=run_this_first
AIRFLOW_CTX_EXECUTION_DATE=2022-05-31T20:09:47.572099+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-31T20:09:47.572099+00:00
[2022-05-31 20:09:54,372] {python.py:175} INFO - Done. Returned value was: None
[2022-05-31 20:09:54,389] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=todoist_load_to_pg, task_id=run_this_first, execution_date=20220531T200947, start_date=20220531T200949, end_date=20220531T200954
[2022-05-31 20:09:54,438] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-31 20:09:54,478] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
